### ISSUE TITLE
JiraConfigServer double slash results in 404 errors

### ISSUE THREAD

**Issue by Yshaidw** [replicaJunction/PSJira#39]
**Recommendation:**
> Please add check for double slash usage on Set-JiraConfigServer :)

After playing arround with errors for a fair bit I discovered a easy to make human error. When using a slash at the end of your Set-JiraConfigServer URL, all scripts fail....of course.. 
Fails: Set-JiraConfigServer "https://test.jira.com/"
Works: Set-JiraConfigServer "https://test.jira.com"

Discovered this when using verbose logging, didn't see it the first couple of times:
`Get-JiraIssue -Key Issue -Verbose`
Returned:
VERBOSE: No Jira sessions have been saved.
VERBOSE: GET http://test.jira.com**//**rest/api/latest/issue/Issue?expand=transitions with 0-byte payload
WARNING: JIRA returned HTTP error 404 - NotFound

```
Set-JiraConfigServer "https://test.jira.com/"
Get-JiraIssue -Key Issue -Verbose

VERBOSE: No Jira sessions have been saved.
VERBOSE: GET http://test.jira.com**//**rest/api/latest/issue/Issue?expand=transitions with 0-byte payload
WARNING: JIRA returned HTTP error 404 - NotFound
The variable cannot be validated because the value  is not a valid value for the Body variable.
At C:\Userpath\WindowsPowerShell\Modules\PSJira\Internal\Invoke-JiraMethod.ps1:88 char:13
+             $body = $readStream.ReadToEnd()
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : MetadataError: (:) [], ValidationMetadataException
    + FullyQualifiedErrorId : ValidateSetFailure
ConvertFrom-Json2 : Cannot bind argument to parameter 'InputObject' because it is an empty string.
At C:\Userpath\WindowsPowerShell\Modules\PSJira\Internal\Invoke-JiraMethod.ps1:91 char:54
+             $result = ConvertFrom-Json2 -InputObject $body
+                                                      ~~~~~
    + CategoryInfo          : InvalidData: (:) [ConvertFrom-Json2], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,ConvertFrom-Json2
```

**Comment by replicaJunction** [comment]
Hi Yshaidw, thanks for the catch! I'm really glad to see that there's enough verbose logging to allow people to notice things like this.

I actually ran across the same issue myself a few days ago, and I came to the same conclusion. Commit 89883ce adds this check for a trailing slash, and that commit is included in the latest release. :)  If you update to the latest version of the module, things should work as expected.
